.github/workflows/linters.yml

```diff
@@ -0,0 +1,45 @@
+name: Linters
+
+on: pull_request
+
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your branch name more descriptive such as `feature/hello-app-frontend` instead of `feature` 👍 Your branch name should tell us what is happening in this pull request and make your pull request more descriptive and professional 😊